content/kubernetes/release-notes/6-4-2-releases/k8s-6-4-2-4.md

```diff
@@ -17,6 +17,11 @@ The Redis Enterprise K8s 6.4.2-4 release supports Redis Enterprise Software 6.4.
 
 The key features, bug fixes, and known limitations are described below.
 
+{{<warning>}} Due to a change in the SCC, on OpenShift clusters running version 6.2.12 or earlier upgrading to version 6.2.18 or later, where `node:1` is <b>not</b> the master node, the upgrade might get stuck.
+
+For more info and steps to prevent this issue, see [upgrade a Redis Enterprise cluster (REC)]({{<relref "/kubernetes/re-clusters/upgrade-redis-cluster.md#before-upgrading">}}).
```

Comment:
```suggestion
For more info and steps to prevent this issue, see [upgrade a Redis Enterprise cluster (REC)]({{<relref "/kubernetes/re-clusters/upgrade-redis-cluster#before-upgrading">}}).
```